tests/fixtures/redis_fixture.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+from pytest_mock_resources import create_redis_fixture
+from redis import Redis
+
+redis = create_redis_fixture()
+
+
+@pytest.fixture(scope="function")
+def redis_setup(redis):
```

Comment:
```suggestion
def redis_client(redis):
```
Additionally I don't see a direct usage of redis_client, do we need this implicitly for `cache.get`?

Reply:
Yes, for `cache` object. Please refer error `ConnectionRefusedError: [Errno 111] Connection refused` [here](https://github.com/RedHatInsights/ros-backend/pull/437#discussion_r1664592633), I was getting inside github actions  while running test that invokes `cache` part from processors. 
In case you have any other suggestion, please let me know. I will try to adapt tests. Thanks!